### PR TITLE
Validate extracted data against schema

### PR DESCRIPTION
**Pull request by KhoomeiK**
part of APE-197

### CODE REVIEW

File: static/schemas.json [hunk 1]
```diff
@@ -565,11 +641,10 @@
...
       "description": "Number of likes, upvotes, etc on the original post"
     },
-    "down_votes": {
-      "type": "integer"
-    },
     "views": {
-      "type": "integer"
+      "type": "integer",
+      "description": "Number of views on the original post",
+      "optional": true
```
Comment (asim-shrestha) [inline static/schemas.json:647]: what this

File: static/schemas.json [hunk 2]
```diff
@@ -565,11 +641,10 @@
       "type": "integer",
       "description": "Number of likes, upvotes, etc on the original post"
     },
-    "down_votes": {
-      "type": "integer"
-    },
```
Comment (asim-shrestha) [inline static/schemas.json:570]: was this not used?

File: static/schemas.json (hunk 1 above)
Comment (KhoomeiK) [reply static/schemas.json]: removed the `optional`

File: static/examples.json [hunk 3]
```diff
@@ -38031,26 +34596,19 @@
...
-            "//philosophy.com/cdn/shop/files/PHB_SG_24_Digital_Visual_Key_Claims_Model_1x1.psd-JPG-300dpi-1550px_743a1a6a-a80a-481a-9ae6-2f335f588fd3.webp?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_SG_macro_1500x1500_c0e1fd3b-b827-41a9-8ba7-5d8537d7a3e3.jpg?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_CPS_SG_Range_16oz_32oz_1x1.psd-JPG-300dpi-1550px.jpg?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_SG_hero_16oz_1500x1500_6a4372d4-3a7c-49ec-b4db-dab71390eac8.jpg?width=1200",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
+            "https://philosophy.com/cdn/shop/files/PHF_AG_24_Digital_CPS_SG_Range_16oz_32oz_1x1.psd-JPG-300dpi-1550px.jpg?width=1200",
```
Comment (asim-shrestha) [inline static/examples.json:34607]: are these the only images though? Might cause problems with the agent if we're not actually retrieving it all

File: static/schemas.json (hunk 2 above)
Comment (KhoomeiK) [reply static/schemas.json:570]: yeah not in any evals

File: static/examples.json (hunk 3 above)
Comment (KhoomeiK) [reply static/examples.json:34607]: yes checked for all of them and it actually correctly retrieves all the images, but it also retrieves duplicates for some of them as well as the base64 placeholder

**Pull request merged**